### ISSUE TITLE
Evaluación existe v/s para todo

### ISSUE THREAD

**Issue by Pipelao** [juanreutter/L-gica2020#21]
Hola @juanreutter 

Tengo una duda con respecto a la evaluación del existe y del para todo que muestro a continuación en la imagen:

![decisiva](https://user-images.githubusercontent.com/26421921/87621239-56437900-c6ee-11ea-8e6c-fc46e83b9f95.PNG)

En este grafo, ¿ evaluacion(phi1, A) y evaluacion(phi2,A) son correctas como las coloqué?

Gracias.

**Comment by juanreutter** [comment]
Correcto. Fijate que la evaluación de phi2 sería vacía igual aunque no estuviese el elemento d. Para que la evaluación de phi2 te devuelva algo, necesitarías pares (a,a) (b,b) y (c,c) en la interpretación de R en A

**Comment by Pipelao** [comment]
Entiendo, muchas gracias @juanreutter
